python/sparkdl/estimators/keras_image_file_estimator.py

```diff
@@ -31,7 +29,137 @@
     keyword_only, HasInputCol, HasOutputCol, SparkDLTypeConverters)
 from sparkdl.transformers.tf_image import TFImageTransformer, OUTPUT_MODES
 import sparkdl.utils.jvmapi as JVMAPI
-import sparkdl.utils.keras_model as keras_utils
+# Uncomment this for testing 
+#import sparkdl.utils.keras_model as keras_utils
+
+logger = logging.getLogger('sparkdl')
+
+##### ##### ##### ##### ##### ##### ##### ##### ##### ##### ##### ##### 
```

Comment:
These are for local REPL based development ~ they will be moved to utility files.